AfsprakenRegels.md

```diff
@@ -781,6 +781,8 @@ Er kan sprake zijn van 1 naam en/of definitie die voor meerdere modelelementen g
 
 Echter, voor bijvoorbeeld voor een relatiesoort of een attribuutsoort kan het ook nodig zijn om dezelfde naam en/of definitie te specificeren. Dit is mogelijk, maar het is dan niet geheel duidelijk of er hetzelfde bedoeld wordt, of dat de (exacte) overeenkomt in een informatiemodel (zoals bv. in een UML informatiemodel) een toevalligheid is. Daarom gelden de volgende afspraken: 
 
+* Als de **uri** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het hetzelfde kenmerk is. Bijvoorbeeld bij objecttype Gebied met attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie" en een objecttype Boom met een attribuutsoort identificatie met _uri_ "http://modellen.geostandaarden.nl/def/nen3610#identificatie".
+
 * Als de **definitie** van twee of meer modelelementen EXACT hetzelfde is, dan wordt hiermee bedoeld dat het dezelfde definitie is. Bijvoorbeeld: attribuutsoort overboeking met _definitie_ "Het bedrag in euro's" en attribuutsoort koopsom met _definitie_ "Het bedrag in euro's".
```

Comment:
Dat lijkt me logisch: definitie = definitie. Maar is het niet dat in dit geval bedoeld wordt dat het modelelement verwijst naar hetzelfde begrip? dus aanpassen in: "dat het modelelement naar hetzelfde begrip verwijst"